packages/storybook/stories/components/combobox/combobox.stories.tsx

```diff
@@ -0,0 +1,381 @@
+        category: CONTROL_CATEGORY.general,
+      },
+      control: 'text',
+    },
+    placeholder: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+      },
+      control: 'text',
+    },
+    readOnly: {
+      table: {
+        category: CONTROL_CATEGORY.general,
+      },
+      control: 'boolean',
+    },
+  }),
+  args: {
+    label: 'My combobox',
+    placeholder: 'Start typing',
+  },
+};
+
+export const Default: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Combobox
+      items={ [
+        { label: 'Dog', value: 'dog' },
+        { label: 'Cat', value: 'cat' },
+        { label: 'Hamster', value: 'hamster' },
+        { label: 'Parrot', value: 'parrot' },
+        { label: 'Spider', value: 'spider' },
+        { label: 'Goldfish', value: 'goldfish' },
+      ] }>
+      <ComboboxControl />
+      <ComboboxContent />
+    </Combobox>
+  ),
+};
+
+export const Overview: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Combobox
+      items={ [
+        { label: 'Dog', value: 'dog' },
+        { label: 'Cat', value: 'cat' },
+        { label: 'Hamster', value: 'hamster' },
+        { label: 'Parrot', value: 'parrot' },
+        { label: 'Spider', value: 'spider' },
+        { label: 'Goldfish', value: 'goldfish' },
+      ] }>
+      <ComboboxControl placeholder={ 'Combobox' } />
+      <ComboboxContent />
+    </Combobox>
+  ),
+};
+
+export const Clearable: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Combobox
+      items={ [
+        { label: 'Dog', value: 'dog' },
+        { label: 'Cat', value: 'cat' },
+        { label: 'Hamster', value: 'hamster' },
+        { label: 'Parrot', value: 'parrot' },
+        { label: 'Spider', value: 'spider' },
+        { label: 'Goldfish', value: 'goldfish' },
+      ] }>
+      <ComboboxControl placeholder={ 'Combobox' } clearable />
+      <ComboboxContent />
+    </Combobox>
+  ),
+};
+
+export const Disabled: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Combobox
+      disabled
+      items={ [
+        { label: 'Dog', value: 'dog' },
+        { label: 'Cat', value: 'cat' },
+        { label: 'Hamster', value: 'hamster' },
+        { label: 'Parrot', value: 'parrot' },
+        { label: 'Spider', value: 'spider' },
+        { label: 'Goldfish', value: 'goldfish' },
+      ] }>
+      <ComboboxControl placeholder={ 'Combobox' } />
+      <ComboboxContent />
+    </Combobox>
+  ),
+};
+
+export const Readonly: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Combobox
+      items={ [
+        { label: 'Dog', value: 'dog' },
+        { label: 'Cat', value: 'cat' },
+        { label: 'Hamster', value: 'hamster' },
+        { label: 'Parrot', value: 'parrot' },
+        { label: 'Spider', value: 'spider' },
+        { label: 'Goldfish', value: 'goldfish' },
+      ] }
+      readOnly>
+      <ComboboxControl placeholder={ 'Combobox' } />
+      <ComboboxContent />
+    </Combobox>
+  ),
+};
+
+export const Group: Story = {
```

Comment:
selecting a value does not work for this example in deployed storybook (tested on FF)